applications/fax/src/fax_worker.erl

```diff
@@ -845,11 +847,6 @@ prepare_contents(JobId, RespHeaders, RespContent) ->
 -spec prepare_contents(kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()) ->
                               {'ok', kz_term:ne_binary()} |
                               {'error', kz_term:ne_binary()}.
-prepare_contents(<<"image/tiff">>, JobId, RespContent, TmpDir) ->
```

Comment:
TIFF files also must be converted to TIFF group 4 format and set to proper resolution